src/assets/styles/StyleConstants.ts

```diff
@@ -1,5 +1,5 @@
 export const Colors = {
-  LIGHT_GREY: '#f5f5f5',
+  LIGHT_GREY: '#f2f2f2',
```

Comment:
Har du sjekket med contrast color check tool? 
Med både svart og blå og rosa knappetekst?

Reply:
Knappene hadde jeg ikke sjekket. Rosafargen funker ikke på ren hvit bakgrunn engang, faktisk. Men siden dette kun skal være gunnvald-vennlig - og kontrasten er fremdeles på 4 - foreslår jeg at det går bra. Ellers må vi finne nye legge inn theme-farger for materialui. Nå bruker vi bare defaults.